borg/crypto.pyx

```diff
@@ -318,6 +329,7 @@ cdef class AES_GCM_256_GMAC:
             if not EVP_DecryptInit_ex(&self.ctx, EVP_aes_256_gcm(), NULL, NULL, NULL):
                 raise Exception('EVP_DecryptInit_ex failed')
             iv = self.fetch_iv(<unsigned char *> idata.buf)
+            self.set_iv(iv)
```

Comment:
needed so that next_iv() works.

we need that after loading / decrypting manifest to know the next IV we may use.